examples/dot-export/README.md

```
@@ -0,0 +1,28 @@
+# DOT Export Example
+
+Minimal, runnable example that exports a DOT graph from `dag.json` and `tasks.json` and renders it to SVG.
+
+
+## Files
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)